src/lib/routing/redirects.test.ts

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, test, vi } from 'vitest';
+import { defaultRedirectStatus, getRedirectTarget, redirectStatusCode } from './redirects';
+
+vi.mock('./redirects.json', () => (
+  {
+    default: [
+      {
+        from: '/redirect-placeholder/:slug/',
+        to: '/en/:slug/',
+        statusCode: '302',
+      },
+      {
+        from: '/redirect-wildcard/*',
+        to: '/en/:splat',
```

Comment:
I removed the trailing slash from the definition in the CMS, because it would result in a double slash in the test scenario for the wildcard match

Reply:
Yup, agreed. Saw the issue 💪